flink-python/README.md

```diff
@@ -10,15 +10,28 @@ In this initial version only Table API is supported, you can find the documentat
 
 ## Installation
 
-Currently, you can install PyFlink from Flink source code. 
-First, you need build the whole Flink project using `mvn clean install -DskipTests` and set the value of the environment variable FLINK_HOME to the `build-target` directory under the root directory of Flink.
-Then enter the directory where this README.md file is located and execute `python setup.py install` to install PyFlink on your device.
+Currently, we can install PyFlink from Flink source code. Enter the directory where this README.md file is located and install PyFlink on your device by executing 
 
-## Running Tests
+```
+python setup.py install
+```
 
-Currently you can perform an end-to-end test of PyFlink in the directory where this file is located with the following command:
+## Running env verify example
```

Comment:
This section seems unnecessary to me. I'm not sure in which case should we check the env manually. When users run ./dev/lint-python.sh, the env will be checked anyway.
Besides, I think that adding one specific test such as test_env_verify_example.py to check the env is not necessary as lint-python.sh will fail if the env is not set correctly.

Reply:
Make sense to me. +1